studio/views.py

```diff
@@ -108,6 +113,84 @@ def profile(request):
     return render(request, "user/profile.html", {"user_profile": user_profile})
 
 
+@login_required
+def delete_account(request):
+    """
+    Renders a form that allows a user to delete their user account.
+    Verifies that the user does not own any Serve projects.
+    """
+    logger.debug("Rendering page delete a user account.")
+    logger.info(f"User views page to delete their user account. User {request.user}")
```

Comment:
Just a heads up. this will print the username/email in the logs which maybe makes the log a bit more sensitive. Should we do that?